test/unit/execution.test.js

```diff
@@ -12,7 +12,7 @@ var _ = require('lodash'),
 describe('execution', function () {
     before(function () {
         execution = new Execution('id', {listen: 'test'}, {}, {});
-        pm = new pmAPI({}, execution, _.noop);
+        pm = new pmAPI(execution, _.noop, _.noop);
```

Comment:
I am assuming these were test setup bugs.

Reply:
pmAPI no longer need the `bridge` instance because assertion events are dispatched using the `onAssertion` function.